learning/web/src/components/CookieBanner/index.js

```diff
@@ -0,0 +1,88 @@
+import Link from "../Link"
+import React, { useEffect } from "react"
+import styled from "styled-components"
+import { mhra, mhra10, mhraGray10 } from "../../utils/colors"
+import { media } from "../../utils/theme"
+
+const StyledCookieBanner = styled.aside`
+  padding: 0 10px 20px;
+  color: ${mhra};
+  background-color: ${mhraGray10};
+
+  div {
+    margin: auto;
+    max-width: 860px;
```

Comment:
we might be able to define this in `sizes` in `../utils/theme`? perhaps? have a think about it. either way I would suggest we set define it in `em`'s. Perhaps you can use this neat trick : `max-width: ${860/ 16}em`